fix: Grammar and typo fixes 2 (#204)
cherryontech/website: src/assets/content/dictionary/SPA.md

sources:
- sourceurl: https://betterprogramming.pub/how-to-implement-a-single-page-application-using-react-router-cc6b9e5c3aec#:~:text=SPA%20stands%20for%20Single%20Page,content%20without%20refreshing%20the%20website.
perspectives:

> - meaning: SPA is used to make the user experience feel a lot smoother. You can tell when it’s a SPA or multi-page application when navigating between menus often because a multi-page application will reload, making the whole UI blink fast depending on the content

Reviewer: ```suggestion
- meaning: a way for me to make the user experience feel a lot smoother. You can tell when it’s a SPA or multi-page application when navigating between menus often because a multi-page application will reload, making the whole UI blink fast depending on the content
```
